_component_design/assisted-search.md

```diff
@@ -44,40 +44,83 @@ When the user clicks the search box, show them a list of keywords that they can
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Layout specifications
+### Active state
 
-Separate groups of keywords using a horizontal divider.
+When the search box is clicked and the cursor is active, a designer may choose to prompt the user with recent or suggested searches depending on their context. Recent Searches is just one example of content that can populate an Assistive Search Box. 
```

Comment:
Any reason for the caps on the Assistive Search Box here and elsewhere?

Reply:
Updated